### DISCUSSION TITLE
Cannot change point size by `size=` in `plotEmbedding()`

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [GreenleafLab/ArchR#1839] [Questions / Documentation] [answered]
I run  `plotEmbedding()` to get the score of a gene in my dataset
Here is my code :
```
p1 <- plotEmbedding(
    ArchRProj = proj, 
    colorBy = "GeneScoreMatrix", 
    name = "mygene", 
    continuousSet = "horizonExtra",
    embedding = "Harmony_ATAC",
    imputeWeights = getImputeWeights(proj),
    size=0.4
)
p1
```
<img width="195" alt="1675867645837" src="https://user-images.githubusercontent.com/81287604/217563486-9cebb624-e36d-4424-8e9f-fbc6b43ffaa3.png">

I changed `size` but nothing happened. 

BTW, when I run the code below to display the clustering, the `size` parameter worked:
```
plotEmbedding(
proj, 
colorBy = "cellColData", 
name = "ClustersHarmonyATAC.0.4", 
embedding = "Harmony_ATAC",  
size = 0.25)
```

**Comment by rcorces** [comment, marked as answer]
Please read all function documentation before posting. Had you read the function documentation for the size parameter, you would see:
<html>
<body>


size | A number indicating the size of the points to plot if plotAs is set to "points".
-- | --



</body>
</html>

Your problem is that the default for `plotAs` is `hex` for GeneScoreMatrix but `points` for `cellColData`

**Comment by jiangpuxuan** [comment]
> plotEmbedding(

Actually I had noticed `plotAs` parameter but misunderstood it.
I am sorry about that and thank you for your help.
